reference/configuration/doctrine.rst

```diff
@@ -499,4 +499,135 @@ can be placed directly under ``doctrine.orm`` config level.
 This shortened version is commonly used in other documentation sections.
 Keep in mind that you can't use both syntaxes at the same time.
 
+Custom Mapping
+--------------
+
+It's possible to not rely on the auto mapping and set all the configuration
+on your own.
+
+For example, you can overwrite the configuration directory for a given bundle.
+Keep in mind that the bundle you want to overwrite has to exist.
```

Comment:
I see now that the example below is actually talking about AppBundle/Resources/config/doctrine, right? If so, we need to change that example to be something different, because I think that's already the default path, right?